x-pack/plugin/esql-core/src/main/java/org/elasticsearch/xpack/esql/core/util/SourceUtils.java

```diff
@@ -21,73 +19,17 @@ public final class SourceUtils {
     private SourceUtils() {}
 
     public static void writeSource(StreamOutput out, Source source) throws IOException {
```

Comment:
Maybe let's mark this as deprecated (for removal)?

Similarly for `readSource`.

Reply:
Looks like there were two copies of `SourceUtils` somehow. One I can just remove. The other I'll deprecate.

Reply:
Oh yeah, the `SourceUtils` in QL was specific for ESQL, we don't need it anymore